### DISCUSSION TITLE
Enhancement Request: Table Mode Icon

### DISCUSSION THREAD

**Discussion by TonyValenti** [josdejong/svelte-jsoneditor#245] [Ideas]
You should put a Table icon next to all arrays to allow them to be opened in table mode.

Something like this:
![image](https://user-images.githubusercontent.com/13609558/229015076-38a31b9b-98a0-4a56-be47-8e041db1199b.png)

**Comment by josdejong** [comment]
Thanks for your suggestion Tony, that is a good idea. We have to balance though not making the view become too crowded, so we'll have to do some experimenting to see if that will work out. 

Note that in `tree` mode, you can already open an array in table mode by right-clicking it, and then selecting "Edit Array" from the context menu. We could do some experimenting with a more prominent button in tree mode too, but same thing there: the view should not get too crowded, I'm not sure.

Anyone interested in experimenting with this idea?
